File: src/main/java/it/unibo/crabinv/Model/save/StartingSaveValues.java
package it.unibo.crabinv.Model.save;

public enum StartingSaveValues {
    LEVEL(1),
    CURRENCY(0);

    private final int value;

    StartingSaveValues(int value) {
        this.value = value;
    }

    public int getIntValue() {
        return value;
    }
}

File: src/main/java/it/unibo/crabinv/model/core/save/StartingSaveValues.kt
package it.unibo.crabinv.model.core.save

/**
 * `enum` of the starting values of the [GameSession]'s [Player].
 * @param intValue the value to associate with the constant
 */
enum class StartingSaveValues(
    /**
     * @return the value of the selected constant.
     */
    val intValue: Int,
) {
    LEVEL(1),
    CURRENCY(0),
    BASE_LEVEL_POWER_UP(1),
}
